### DISCUSSION TITLE
refound

### DISCUSSION THREAD

**Discussion by josh21kurumi** [aclap-dev/video-downloadhelper#1474] [Help and Q&A]
### Your question

can i have a refund for my firefox version ? From my Chrome extension, I wanted to purchase the premium version of Video DownloadHelper. I clicked, but it redirected me to a Firefox version, which I bought, thinking it would be usable for both browsers. However, I ended up with a Firefox version that is unusable for Chrome. I have since found a Chrome version, but I would like a refund for my Firefox version.

### Addon & CoApp version and details

From my Chrome extension, I wanted to purchase the premium version of Video DownloadHelper. I clicked, but it redirected me to a Firefox version, which I bought, thinking it would be usable for both browsers. However, I ended up with a Firefox version that is unusable for Chrome. I have since found a Chrome version, but I would like a refund for my Firefox version.

**Comment by jcfrog** [comment]
Hi,
please send us some details at licenses@downloadhelper.net
